src/snowcli/cli/snowpark/procedure_coverage/commands.py

```diff
@@ -42,10 +46,13 @@ def procedure_coverage_report(
     store_as_comment: bool = StoreAsCommandOption,
     **options,
 ):
+    snowpark = get_snowpark_project_definition()
     message = ProcedureCoverageManager().report(
         identifier=identifier,
         output_format=output_format,
         store_as_comment=store_as_comment,
+        app_stage_path=get_app_stage_path(snowpark),
+        artefact_name="app.zip",
```

Comment:
I don't think I understand why app.zip is hardcoded here?

Reply:
Fixed